src/android/Library/src/MultiImageChooserActivity.java

```diff
@@ -456,21 +492,7 @@ public View getView(int pos, View convertView, ViewGroup parent) {
 
             final int id = imagecursor.getInt(image_column_index);
             final int rotate = imagecursor.getInt(image_column_orientation);
-            if (isChecked(pos)) {
-                if (android.os.Build.VERSION.SDK_INT>=16) {
-                  imageView.setImageAlpha(128);
-                } else {
-                  imageView.setAlpha(128);	
-                }
-                imageView.setBackgroundColor(selectedColor);
-            } else {
-                if (android.os.Build.VERSION.SDK_INT>=16) {
-                  imageView.setImageAlpha(255);
-                } else {
-                  imageView.setAlpha(255);	
-                }
-                imageView.setBackgroundColor(Color.TRANSPARENT);
-            }
```

Comment:
I didn't notice the image shrinking in the screenshot. This plugin currently crashes in the play store version of Zenput if you rotate the device after pressing the OK button once you've selecting images. I say we come back to this later.